docker-compose/core-valid/docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
     image: farao/gridcapa-core-valid:latest
     volumes:
       - ./config/core-valid-runner-application.yml:/config/application.yml
-      - ../../configuration/test/core-valid-runner-itools-config.yml:/home/farao/.itools/config.yml
```

Comment:
did you check for other /test/ itools in other projects ?(other than core cc and valid ?)